packaging/dbscripts/clusters_sp.sql

```diff
@@ -583,7 +610,7 @@ CREATE OR REPLACE FUNCTION ClearClusterUpgradeRunning (
 AS $PROCEDURE$
 BEGIN
     UPDATE cluster
-    SET upgrade_running = false
+    SET upgrade_running = false, upgrade_progress = 0, upgrade_correlation_id = NULL
```

Comment:
Please reformat to have one column update per line

Reply:
> Please reformat to have one column update per line

Ack

> Not sure if we should clear the upgrade_correlation_id at this point, as already mentioned above, this could be used to provide info about last upgrade

That's reasonable